gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/EntityService.java

```diff
@@ -1,11 +1,13 @@
 package org.hypertrace.gateway.service.entity;
 
-import com.codahale.metrics.Timer;
```

Comment:
yeees! can you remove the dependency on this as well in build.gradle.kts

Reply:
Done